File: python/integracion_con_bases_de_datos/guardar_un_contacto_basico/conexion.py
import mysql.connector
def conectar():
    try:
        conexion = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",  # en XAMPP normalmente está vacío
            database="guardar_contacto_basico"
        )
        return conexion
    except mysql.connector.Error as e:
        print("Error al conectar:", e)
        return None

File: visual_basic/integracion_con_base_de_datos/buscar_contacto_por_nombre/conexion.vb
Imports MySql.Data.MySqlClient
Module conexion
    Public Function ObtenerConexion() As MySqlConnection
        Dim cadenaConexion As String = "server=localhost;user id=root;password=;database=buscar_el_contacto_por_nombre"
        Return New MySqlConnection(cadenaConexion)
    End Function
End Module